apps/api/src/schema/lib/day.ts

```diff
@@ -80,5 +80,6 @@ const daysTransform = (days: string | undefined, ctx: z.RefinementCtx) => {
 };
 
 export const daysSchema = z.string().transform(daysTransform).openapi({
+  description: "The days of the week for when the course meets",
```

Comment:
This is not quite right. Try this endpoint yourself and see what this day parameter does. The way it's worded also seems to describe a course returned by the endpoint, whereas you are describing a filter \*parameter* on the query which could apply to any number of returned objects.

I also assume you've already verified that this description is correct for LARC queries as well, the other place where this schema is used.